torch/_inductor/scheduler.py

```diff
@@ -1144,6 +1144,10 @@ def create_backend(self, device: torch.device):
             from .codegen.cpp import CppScheduling
 
             return CppScheduling(self)
+        elif device.type == "mps":
+            raise RuntimeError(
+                f"Device type {device.type} is not supported yet with the Inductor compiler"
```

Comment:
Should we recommend `aot_eager` here?
Or should we swap the default on MPS to be aot_eager instead of inductor?

Reply:
Will let @kulinseth to make the final decision on this one.